roles/openshift_logging_curator/tasks/main.yaml

```
@@ -33,13 +33,17 @@
     name: "aggregated-logging-curator"
     namespace: "{{ openshift_logging_namespace }}"
     image_pull_secrets: "{{ openshift_logging_curator_image_pull_secret }}"
+    secrets:
+    - logging-curator
```

```suggestion
      - logging-curator
```
to fix `48:5      error    wrong indentation: expected 2 but found 4  (indentation)`